src/Propulsion.DynamoStore/DynamoStoreSource.fs

```diff
@@ -116,10 +116,9 @@ module private Impl =
 [<NoComparison; NoEquality>]
 type LoadMode =
     /// Skip loading of Data/Meta for events; this is the most efficient mode as it means the Source only needs to read from the index
-    | WithoutEventBodies of categoryFilter : (string -> bool)
-    /// Populates the Data/Meta fields for events; necessitates loads of all individual streams that pass the categoryFilter before they can be handled
-    | Hydrated of categoryFilter : (string -> bool)
-                  * degreeOfParallelism : int
+    | IndexOnly
```

Comment:
Maybe it should be called `HydrationMode` (or Strategy?), this case renamed to `IndexedFieldsOnly` and the other one `Full` ?

Reply:
Changed to type `EventHydration = IndexedFieldsOnly | HydrateBodies`
Not sure I'm happy yet - Hydrate probably looks better
Also not sure what to call loadMode now (renamed it to `mode`, but maybe it should be called `hydration` or `events` given the type and the case both have `Hydrate` in it. Hm, will let it settle, but open to ideas..

Reply:
(resolved out of band with suggestions from Einar 🙏 )